src/Documentation/sidebar.json

```diff
@@ -115,10 +115,79 @@
       "dvcignore",
       "updating-tracked-files",
       "large-dataset-optimization",
-      "external-dependencies",
       {
-        "label": "Managing External Data",
```

Comment:
I still don't see a single reason to do this. HTTP is an edge case that should hardly affect this decision.